client/package.json

```diff
@@ -66,6 +67,10 @@
     "eslint-plugin-only-warn": "^1.0.2",
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
+    "jest-localstorage-mock": "^2.4.9",
```

Comment:
It seems to me that this is not used in the code anymore

Reply:
Just found a point where it is still used, but please have a look at the comment I left there